src/ducks/modules/sessions.js

```diff
@@ -108,14 +107,14 @@ const addNodes = (nodes, additionalAttributes) => (dispatch, getState) => {
   });
 };
 
-const updateNode = (node, full = false) => (dispatch, getState) => {
+const updateNode = (node, additionalAttributes = false) => (dispatch, getState) => {
```

Comment:
Maybe this would be a good point to separate this action into something like updateNodeAttributes:

`updateNodeAttributes = (id, attributes) =>`